### PR TITLE
add a region-based constructor which uses the aws-region crate

### PR DESCRIPTION
**Pull request by njaard**
# Pull Request

## Related issue
Does not fix an issue

## What does this PR do?
- Makes it so you don't need to determine the URL if you only know the name of the AWS region

## PR checklist
Please check if your PR fulfills the following requirements:
- [ ] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [ ] Have you read the contributing guidelines?
- [×] Have you made sure that the title is accurate and descriptive of the changes?

### CODE REVIEW

File: src/client.rs [hunk 1]
```diff
@@ -35,6 +35,9 @@ impl Client {
     pub fn builder(url: impl AsRef<str>) -> Result<Builder<MissingCred>> {
         Builder::new(url)
     }
+    pub fn build_region(url: impl AsRef<str>) -> Result<Builder<MissingCred>> {
+        Builder::new(url)
+    }
```
Comment (irevoire) [inline src/client.rs]: I guess we were supposed to take an AWS region in parameter here + it’s missing the doctest

**Review by irevoire — CHANGES_REQUESTED** [review]
Hey, thanks for your contribution, I think you forgot to update one last function

**Comment by njaard** [comment]
7 months later, but I fixed it!

I also improved the doctests for the other function

**Review by irevoire — APPROVED** [review]
Awesome! Thanks a lot @njaard 🫶 
Are you using strois in one of your project?

bors merge

**Pull request merged**